### PR TITLE
chore(cloudfront): migrate edge function handler

### PR DESCRIPTION
**Pull request by colifran**
This PR moves the cloudfront edge function handler from aws-cdk-lib to our new centralized location for custom resource handlers in the @aws-cdk package.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by vinayak-kukreja** [comment]
@Mergifyio update

File: yarn.lock [hunk 1]
```diff
@@ -1214,6 +1214,53 @@
     tslib "^2.5.0"
     uuid "^8.3.2"
 
+"@aws-sdk/client-ssm@^3.441.0":
```
Comment (vinayak-kukreja) [inline yarn.lock:1217]: Could we remove the `^` and fix the sdk version?

**Review by vinayak-kukreja — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (vinayak-kukreja) [reply yarn.lock:1217]: And align it with the version we have across the repo: `3.421.0`.

Comment (colifran) [reply yarn.lock]: Updated!

**Review by vinayak-kukreja — APPROVED** [review]

**Pull request merged**